docs/pages/enroll-resources/desktop-access/dynamic-registration.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Dynamic Windows Desktop Registration
+description: Register/unregister Windows desktops without restarting Teleport.
+---
+
+Dynamic Windows desktop registration allows Teleport administrators to register new Windows desktops (or
+update/unregister existing ones) without having to update the static
+configuration files read by Teleport Windows Desktop Service instances.
+
+Windows Desktop Service instances periodically query the Teleport Auth Service for
+`dynamic_windows_desktop` resources, each of which includes the information that the Windows Desktop
+Service needs to connect to a Windows desktop.
+
+## Required permissions
+
+In order to interact with dynamically registered Windows desktops, a user must have
+a Teleport role with permissions to manage `dynamic_windows_desktop` resources.
```

Comment:
Do any of the preset roles have this permission by default, or do we always have to add it?

Is this behavior the same as for apps/databases?